cityelf/src/main/java/com/cityelf/repository/NotificationRepository.java

```diff
@@ -2,12 +2,22 @@
 
 import com.cityelf.model.Notification;
 import com.cityelf.model.User;
+
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public class NotificationRepository {
 
-  public Notification getNotification(User user) {
-    return new Notification();
+  @Autowired
+  private UserRepository userRepository;
+
+  public Notification getNotification(long id) {
+    User innerUser = userRepository.getUser(id);
+    if (innerUser != null) {
+      return innerUser.getNotification();
+    } else {
+      return null;
```

Comment:
if you return **null** from the method, you should consider Java 8 Optional class